packages/ui/package.json

```diff
@@ -41,7 +41,7 @@
     "lucide-react": "0.447.0",
     "react": "19.0.0",
     "react-dom": "19.0.0",
-    "react-shadow-scope": "1.0.11",
+    "react-shadow-scope": "1.0.5",
```

Comment:
Was seeing a weird error with the context menu, so I've downgraded this to a known working version.